src/lightning/pytorch/trainer/connectors/logger_connector/result.py

```diff
@@ -357,6 +357,24 @@ def log(
         rank_zero_only: bool = False,
     ) -> None:
         """See :meth:`~lightning.pytorch.core.module.LightningModule.log`"""
+        # if value is a Metric and sync_dist != None, then
```

Comment:
These comments just duplicate what was written in code. Comments should describe the "why", not the "what"

Reply:
Changed this. Can you maybe check now?